.github/scripts/package_lfc.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd $GITHUB_WORKSPACE
+
+outname="lfc_nightly_$(date '+%Y%m%d-%H%M%S')"
+
+# build lfc compiler
+./gradlew generateStandaloneCompiler
+
+# assemble the files in a separate directory
+mkdir -p $outname/bin
+mkdir -p $outname/lib
+
+mv org.lflang/build/libs/org.lflang-0.1.0-SNAPSHOT-all.jar $outname/lib
```

Comment:
Should this better be `org.lflang/build/libs/org.lflang-*-SNAPSHOT-all.jar` to account for future version updates? I think the yml file below does something similar for the RCA artifact.

Reply:
Good point, I will fix that.